src/main/webapp/app/exercises/programming/shared/commits-info/commits-info-group/commits-info-row/commits-info-row.component.html

```diff
@@ -0,0 +1,74 @@
+<tr class="{{ isExpanded ? (groupCommitIndex === groupCommitCount - 1 ? 'bg-light border-bottom' : 'bg-light') : '' }}">
```

Comment:
```suggestion
<tr [class.bg-light]="isExpanded" [class.border-bottom]="groupCommitIndex === groupCommitCount - 1">
```
You might need to also include `isExpanded` in the second condition, I'm not 100% familiar with your code